python/test/unit/conftest.py

```diff
@@ -20,6 +20,51 @@ def pytest_addoption(parser):
         parser.addoption("--forked", action="store_true")
 
 
+def pytest_pyfunc_call(pyfuncitem: pytest.Function):
+
+    if os.name == "nt" and "forked" in pyfuncitem.keywords:
+        # Avoid recursion
+        if os.getenv("_PYTEST_SUBPROCESS_RUNNING"):
+            return None
+
+        import subprocess
+
+        pos = pyfuncitem.nodeid.find(str(pyfuncitem.path.relative_to(pathlib.Path(os.getcwd()))))
```

Comment:
Do we need `os.getcwd()`? I think `pathlib.Path()` should work for the current directory (please double check)
```suggestion
        pos = pyfuncitem.nodeid.find(str(pyfuncitem.path.relative_to(pathlib.Path())))
```

Reply:
I believe we need `os.getcwd`:
```python
>>> import pathlib
>>> import os
>>> pathlib.Path()
WindowsPath('.')
>>> os.getcwd()
'C:\\projects\\intel-xpu-backend-for-triton\\test_forked'
```

Reply:
I think `.` and `C:\\projects\\intel-xpu-backend-for-triton\\test_forked` is the same directory in this case). I agree let's keep `os.getcwd` for clarity.